README.md

```diff
@@ -132,14 +131,27 @@ cinatra是header-only的，直接引用头文件既可。
 			return true;
 		}
 	};
-	
+
+	//将信息从中间件传输到处理程序
```

Comment:
Please use english comment. 
BTW, do you know Chinese?

Reply:
Nope :D It was an attempt with google translate. Hope it didn't get too wrong :>
I placed the english comment on the lang/english/README.md